graphql2/graphqlapp/config.go

```diff
@@ -4,19 +4,26 @@ import (
 	"context"
 
 	"github.com/target/goalert/config"
+	"github.com/target/goalert/expflag"
 	"github.com/target/goalert/graphql2"
 	"github.com/target/goalert/permission"
 )
 
 func (a *Query) IntegrationKeyTypes(ctx context.Context) ([]graphql2.IntegrationKeyTypeInfo, error) {
 	cfg := config.FromContext(ctx)
-	return []graphql2.IntegrationKeyTypeInfo{
+	typeInfo := []graphql2.IntegrationKeyTypeInfo{
 		{ID: "email", Name: "Email", Label: "Email Address", Enabled: cfg.EmailIngressEnabled()},
 		{ID: "generic", Name: "Generic API", Label: "Generic Webhook URL", Enabled: true},
 		{ID: "grafana", Name: "Grafana", Label: "Grafana Webhook URL", Enabled: true},
 		{ID: "site24x7", Name: "Site 24x7", Label: "Site24x7 Webhook URL", Enabled: true},
 		{ID: "prometheusAlertmanager", Label: "Alertmanager Webhook URL", Name: "Prometheus Alertmanager", Enabled: true},
-	}, nil
+	}
+
+	if expflag.ContextHas(ctx, expflag.UnivKeys) {
+		typeInfo = append(typeInfo, graphql2.IntegrationKeyTypeInfo{ID: "universal", Label: "Universal Integration Key", Name: "Universal Integration Key", Enabled: true})
```

Comment:
These field names aren't great, but "Label" is for the copy link
```suggestion
		typeInfo = append(typeInfo, graphql2.IntegrationKeyTypeInfo{ID: "universal", Label: "Universal Integration Key URL", Name: "Universal Integration Key", Enabled: true})
```